pom.xml

```
@@ -6,14 +6,73 @@
     <groupId>fish.payara.advisor</groupId>
     <artifactId>advisor-maven-plugin</artifactId>
     <packaging>maven-plugin</packaging>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0</version>
+    <name>${project.groupId}:${project.artifactId}</name>
+    <description>Payara Advisor Tool</description>
+    <url>http://www.payara.fish</url>
```

Don't this this is necessary?
```suggestion
```

---

based on the issue I saw from logs, yes this is needed. This is the message:
`[ERROR]     * Invalid POM: /fish/payara/advisor/advisor-maven-plugin/1.0/advisor-maven-plugin-1.0.pom: Project name missing, Project description missing, Project URL missing, License information missing, SCM URL missing, Developer information missing
[ERROR]`

---

In which case it should link to this repo rather than our website:
```suggestion
    <url>https://github.com/payara/AdvisorTool/</url>
```

---

already updated, thanks